contracts/PushCore/PushCoreV3.sol

```diff
@@ -768,21 +763,22 @@ contract PushCoreV3 is
     }
 
     /// @inheritdoc IPushCoreV3
-    function handleChatRequestData(address requestSender, address requestReceiver, uint256 amount) public {
+    function handleChatRequestData(address requestSender, address requestReceiver, uint256 amount) external {
```

Comment:
Name of the external function is inadequate.

`handleChatRequestData() `should be renamed to `createIncentivizedChatRequest()`